src/App.tsx

```diff
@@ -10,33 +11,22 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [id, setId] = useState(tabs[0].id);
+
+  const setTab = (tabId: string) => {
+    setId(tabId);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is Tab ${id.slice(-1)}`}
```

Comment:
Because of it you could have fallen tests. Check with this option.
```suggestion
        {`Selected tab is ${selectedTab.title}`}
```